etc/profile-m-z/trojita.profile

```diff
@@ -57,7 +57,10 @@ private-dev
 private-etc alternatives,ca-certificates,crypto-policies,fonts,hostname,hosts,pki,resolv.conf,selinux,ssl,xdg
 private-tmp
 
-dbus-user none
+dbus-user filter
+dbus-user.talk org.freedesktop.Notifications
```

Comment:
`org.freedesktop.Notifications` is insecure (allows sandbox escapes, code execution,  screenshots, keylogger, ...). It should be avoided if the program works w/o it and the profile is not complete open.

Reply:
It works but won't display the notification on tray I think. 
I can remove it make it like this :

```
dbus-user none
# Uncomment both to allow keyring access 
# dbus-user filter
# Uncomment to use system keyring
# dbus-user.talk org.freedesktop.secrets
dbus-system none
```
Not sure how to resolve it; just don't want it to have a secrets access for no reason if the user chooses plain-text storage.